src/Module/Proxy.php

```diff
@@ -131,7 +131,9 @@ public static function init(array $parameters = [])
 		// Store original image
 		if ($direct_cache) {
 			// direct cache , store under ./proxy/
-			file_put_contents($basepath . '/proxy/' . ProxyUtils::proxifyUrl($request['url'], true), $image->asString());
+			$filename = $basepath . '/proxy/' . ProxyUtils::proxifyUrl($request['url'], true);
+			file_put_contents($filename, $image->asString());
+			chmod($filename, DI::config()->get('system', 'proxy_file_chmod'));
```

Comment:
I, personally, would like to have this behaviour configurable. Means: It should do nothing by default and only should act when activated.

Reply:
Isn't that a a bit over-complicating the code? I mean, nothing changes for you, just a `chmod()` call.

Reply:
Well. It does change something. It performs a `chmod` on my system to some value that previously had been set by my machine.

And: What happens when someone sets the value to some empty string?